docs/APPLE_SIGNIN_SETUP.md

```
@@ -33,355 +41,136 @@ Apple Sign In uses a **native authentication flow** on iOS, which differs from t
-
-Apple OAuth secret keys expire every **6 months**. Supabase shows a warning about this:
-
-> ⚠️ Apple OAuth secret keys expire every 6 months. A new secret should be generated every 6 months, otherwise users on the web will not be able to sign in.
+| Field                            | Value                         | Notes                                                  |
+| -------------------------------- | ----------------------------- | ------------------------------------------------------ |
+| **Client IDs**                   | `com.volvox.sobrietywaypoint` | Your iOS bundle ID                                     |
+| **Secret Key (for OAuth)**       | _(leave empty)_               | Not needed for native iOS flow                         |
+| **Allow users without an email** | Off                           | Toggle on only if you want to allow hidden email users |
 
-**Best practices:**
+5. Click **Save**
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)